src/options.cpp

```diff
@@ -1514,6 +1514,17 @@ void options_manager::add_options_graphics()
        );
 #endif
 
+    add( "RENDERER", "graphics", translate_marker( "Renderer" ),
+         translate_marker( "Set which renderer to use.  Requires restart." ),
+#if (defined TILES)
+         // No renderer selection in non-TILES mode
+         cata_tiles::build_renderer_list(),
+#else
+    {   { "software", translate_marker( "software" ) }
+    },
+#endif
+         "software", COPT_NO_HIDE );
+
     add( "SOFTWARE_RENDERING", "graphics", translate_marker( "Software rendering" ),
```

Comment:
We should get rid of this option since it's redundant with renderer=software

Reply:
Done.